src/main/java/uk/gov/hmcts/darts/common/entity/Hearing.java

```diff
@@ -62,4 +66,7 @@ public class Hearing {
         inverseJoinColumns = {@JoinColumn(name = "moj_eve_id")})
     private List<Event> theEvents;
 
+    @ManyToMany(mappedBy = "hearings")
```

Comment:
i think we have changed the design now, so that a hearing is only associated with 1 case